### ISSUE TITLE
Refined UAVDT

### ISSUE THREAD

**Issue by ydhcg-BoBo** [Songinpyo/SFTrack#1]
As mentioned in your manuscript, could you please share when the enhanced dataset will be available for future research? Thank you.

**Comment by Songinpyo** [comment]
Thanks for your interest!
We gonna make public the enhanced annotations in few days. :)

**Comment by Songinpyo** [comment]
We uploaded the refined annotations!

**Comment by ydhcg-BoBo** [comment]
Thank you

**Comment by ydhcg-BoBo** [comment]
Thank you for your work. You have only provided the refined annotations for the test set. Could you please provide the refined annotations for the training set for the sake of subsequent research and fair comparison?
